rollup.config.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const babel = require('rollup-plugin-babel');
+const readPkgUp = require('read-pkg-up');
+const resolve = require('rollup-plugin-node-resolve');
+const json = require('rollup-plugin-json');
+const commonjs = require('rollup-plugin-commonjs');
+const postcss = require('rollup-plugin-postcss');
+const peerDeps = require('rollup-plugin-peer-deps-external');
+const getBabelPreset = require('@commercetools-frontend/babel-preset-mc-app');
+const builtins = require('rollup-plugin-node-builtins');
+const babelPluginImportGraphQL = require('babel-plugin-import-graphql');
+const postcssCustomProperties = require('postcss-custom-properties');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const postcssColorModFunction = require('postcss-color-mod-function');
+const postcssDiscardComments = require('postcss-discard-comments');
+const postcssImport = require('postcss-import');
+const postcssPresetEnv = require('postcss-preset-env');
+const postcssReporter = require('postcss-reporter');
+
+const { pkg } = readPkgUp.sync({
+  cwd: fs.realpathSync(process.cwd()),
+});
+
+const format = process.env.npm_lifecycle_event.split(':')[1];
+
+const babelOptions = getBabelPreset();
+
+// Inspired by https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.prod.js
+const browserslist = {
+  production: [
+    '>1%',
+    'last 2 versions',
+    'Firefox ESR',
+    'not op_mini all',
+    'ie 11',
+  ],
+};
+
+const config = {
+  output: {
+    name: pkg.name,
+    sourcemap: true,
+    globals: {
+      react: 'React',
+      redux: 'redux',
+      'react-redux': 'react-redux',
```

Comment:
Why do they need to be globals if we exclude deps from being bundled after all. I think I knew why once but can‘t recall.